style/style.css

```diff
@@ -0,0 +1,165 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+  margin-top: -40px;
+  margin-left: 0;
+  margin-right: 0;
+  cursor: pointer;
+  width: 100%;
+  justify-content: center;
+}
+
+.nav-bar {
+  position: relative;
+  display: flex;
+  flex: 1 1 11.11%;
+  top: 130px;
+  width: 990px;
+  height: 40px;
+  border-radius: 3px;
+  box-shadow: rgba(56, 56, 56, 1) 0 1px 2px;
+  background: linear-gradient(to top, rgb(120, 120, 120) 17px, rgb(100, 100, 100) 17px);
+  align-items: stretch;
+  z-index: 1;
+}
+
+.nav-bar > .btn-item {
+  width: 11.12%;
+  height: 35px;
+  margin: 0 -1px;
+  background-color: transparent;
+  border: none;
+  border-right: 1px solid rgb(66, 66, 66);
+  color: white;
+  font-weight: 400;
+}
+
+.nav-bar > .btn-item:hover {
+  background: linear-gradient(#000, #444, #444, #444, #000);
+}
+
+a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  width: 10%;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_